armory/__init__.py

```diff
@@ -2,20 +2,29 @@
 
 # Set up logging for a library
 import logging
+import sys
 
 logger = logging.getLogger(__name__)
 logger.addHandler(logging.NullHandler())
 
 # Submodule imports
-from armory import art_experimental
-from armory import baseline_models
-from armory import data
-from armory import docker
-from armory import eval
-from armory import paths
-from armory import scenarios
-from armory import utils
-from armory import webapi
+try:
+    import coloredlogs
+
+    from armory import art_experimental
+    from armory import baseline_models
+    from armory import data
+    from armory import docker
+    from armory import eval
+    from armory import paths
+    from armory import scenarios
+    from armory import utils
+    from armory import webapi
+except ImportError as e:
+    module = e.name
+    print(f"ERROR: cannot import '{module}' module", file=sys.stderr)
+    print(f"    Please run: $ pip install -r requirements.txt", file=sys.stderr)
+    sys.exit(1)
```

Comment:
I think we can just return an error here instead of a sys import and sys.exit?

Reply:
changed to `raise`.